src/d3a/d3a_core/simulation.py

```diff
@@ -286,6 +288,7 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
         tick_lengths_s = config.tick_length.total_seconds()
         slot_count = int(config.sim_duration / config.slot_length)
 
+        self.simulation_config.external_redis_communicator.sub_to_aggregator(aggregator)
```

Comment:
Can you point me from where is argument `aggregator ` coming from ?

Reply:
https://github.com/gridsingularity/d3a/pull/798/files/82a65e57e35ecb71dd06e43c888007f2ee81346c#diff-fa25e2850120fece94bc9fbdfbd1266eR50